corehq/apps/settings/views.py

```diff
@@ -299,7 +299,7 @@ def page_context(self):
     def post(self, request, *args, **kwargs):
         if self.password_change_form.is_valid():
             self.password_change_form.save()
-            messages.success(request, _("Your password was successfully changed!"))
+            messages.success(request, _("Your password was successfully changed."))
```

Comment:
I don't think you should change this one, since it will require a re-translation.

Reply:
fixed: https://github.com/dimagi/commcare-hq/pull/19574/commits/ef3121686456ffd8e920d6b4284058b74edf7681